torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -124,18 +135,32 @@ class PytorchBotHandler {
   async handleMerge(
     forceMessage: string,
     mergeOnGreen: boolean,
-    landChecks: boolean
+    landChecks: boolean,
+    landChecksEnrolled: boolean
   ) {
     const isForced = forceMessage != undefined;
     const isValidMessage = this.isValidForceMergeMessage(forceMessage);
+    const extra_data = {
+      forceMessage,
+      mergeOnGreen,
+      landChecks,
+      landChecksEnrolled,
+    };
 
     if (!isForced || isValidMessage) {
-      await this.dispatchEvent("try-merge", isForced, mergeOnGreen, landChecks);
+      await this.logger.log("merge", extra_data);
```

Comment:
Lets also include the PR number, the user who initiated the merge command, and perhaps even comment number so that we can tell which merge command prob actually resulted in a successful outcome (this could be done in the logger itself) so that people querying rockset don't have to decode the dynamoKey